NitroxClient/Communication/Packets/Processors/BroadcastEscapePodsProcessor.cs

```diff
@@ -68,10 +68,8 @@ private void AssignPlayerToEscapePod(BroadcastEscapePods packet)
 
         // Done in a coroutine because we want to wait until the escape pod has fully loaded
         // at the beginning of the game.
-        private IEnumerator SyncEscapePodGuids(BroadcastEscapePods packet)
+        private void SyncEscapePodGuids(BroadcastEscapePods packet)
         {
-            yield return new WaitUntil(() => EscapePod.main.gameObject);
```

Comment:
Why was this removed?  Do we not want to wait until the gameobject is loaded?

Reply:
It didn't seem to be needed and I removed it because it only let the code continue after the InitialPlayerSync so the items could not be placed in it. I'll try to restore it and see if it works that way.